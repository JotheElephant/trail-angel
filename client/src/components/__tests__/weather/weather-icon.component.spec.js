'use strict';

import { View } from 'react-native';
import React from 'react';

import WeatherIcon from '../../weather/weather-icon.component';
import weatherData from '../../../../__tests__/fixtures/weather-data';

describe('weather icon component', () => {
  it('renders correctly', () => {
    const tree = (
      <View>
        <WeatherIcon icon={weatherData.daily.data[0].icon} />
        <WeatherIcon icon={weatherData.daily.data[1].icon} />
        <WeatherIcon icon={weatherData.daily.data[2].icon} />
      </View>
    );
    expect(tree).toMatchSnapshot();
  });
});