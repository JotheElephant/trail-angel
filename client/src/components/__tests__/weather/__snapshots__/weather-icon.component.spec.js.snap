exports[`weather icon component renders correctly 1`] = `
<View>
  <WeatherIcon
    icon="snow" />
  <WeatherIcon
    icon="snow" />
  <WeatherIcon
    icon="rain" />
</View>
`;
